pkg/retry/retry_test.go

```diff
@@ -105,7 +105,7 @@ func TestRetrier_RetryWithoutBackoff(t *testing.T) {
 			name: "a retry executions without backoff should be at constant rate. (10ms, 4 retries)",
 			cfg: retry.Config{
 				MaxTries:      4,
-				EnableBackoff: true,
```

Comment:
this should be true according to the config right? otherwise the test is not valid

Reply:
This is the test without backoff, if we provide `EnableBackoff: true` retry time will not be constant. 
According to the test name this should be without backoff.

Reply:
oh right sure, I think I misread the header